kubernetes/samples/scripts/create-rcu-schema/README.md

```diff
@@ -0,0 +1,200 @@
+# Creating Oracle DB service and RCU Schema for a Fusion Middleware domain
+
+This sample demonstrates how to create an Oracle DB service on Kubernetes cluster and create RCU schema on the Oracle DB being used by a Fusion Middleware domain.  
+
+The directory contains the following sample scripts to start an Oracle DB service in default namespace, stop the Oracle DB service, create an RCU schema and drop the RCU schema
+
+```
+$ ./start-db-service.sh   
```

Comment:
there is no explanation of what this is, what directory it is in, etc. ???

Reply:
surely it needs some parameters or an imagePullSecret or something? you need to authenticate to pull that image don't you?

Reply:
The scripts in the new directory create-rcu-schema is based on the document as described  in https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/fmw-infra/

Here we create convenient scripts out of the above document so that user has not to cut-n-paste the document every-time.  I copied the db service/deployment yaml file from this document (which does not have imagePullSecret)   The pre-requisite is that customer has to docker login into container-registry.oracle.com and download DB-slim and  fmw-infra images. I can put this information on the main page w/o adding imagePullSecret in oradb.yaml

Alternatively I can check the result for docker pull  command and provide the suggested action as follows  e.g. in  start-db-service.sh
===
ocr_pfx=container-registry.oracle.com
db_image=${ocr_pfx}/database/enterprise:12.2.0.1-slim
docker pull ${db_image}
if [ $? != 0  ]; then
 echo "######################";
 echo "[ERROR] Could not pull ${db_image}";
 echo "Please run  [ docker login ${ocr_pfx} ]  and "
 echo "Check-out the db image (if needed)"
 echo "######################";
 exit -1;
fi  
===